docs/pages/about.tsx

```diff
@@ -490,7 +490,7 @@ const emeriti = [
   {
     name: 'Dmitriy Kovalenko',
     github: 'dmtrKovalenko',
-    twitter: 'dmtrKovalenko',
+    twitter: 'goose_plus_plus',
```

Comment:
@dmtrKovalenko My heart is with you and the Ukrainian people.

Reply:
Thank you 🙏